configs/approval.json

```diff
@@ -113,44 +107,160 @@
           "resourceDefinitions": [
             {
               "attributeFilter": {
-                "value": "*",
-                "key": "id",
-                "operation": "equal"
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
               }
             }
           ]
         },
         {
-          "permission": "approval:actions:create",
+          "permission": "approval:taglinks:create",
```

Comment:
@hsong-rh are you exposing the tag links or are you going to control it from workflows?

Reply:
@mkanoor Changed to control from workflows